src/course/en/articles.json

```diff
@@ -37,24 +39,38 @@
             "_banks": {
                 "_isEnabled": true,
                 "_split": "2,2",
-                "_randomisation": true
+                "_randomisation": false
             },
             "_questions": {
                 "_resetType": "hard",
                 "_canShowFeedback": false,
                 "_canShowMarking": false,
                 "_canShowModelAnswer": false
             },
-            "_isResetOnRevisit": true,
+            "_isResetOnRevisit": false,
             "_attempts": "infinite"
         },
         "_trickle": {
             "_isEnabled": true,
             "_button": {
                 "_isEnabled": true,
+                "_styleBeforeCompletion": "visible",
+                "_comment1": "hidden | visible",
+                "_styleAfterClick": "disabled",
+                "_comment2": "hidden | disabled | scroll",
+                "_isFullWidth": false,
+                "_autoHide": false,
+                "_className": "",
+                "_hasIcon": true,
                 "text": "Continue",
-                "startText": "Continue",
-                "finalText": "Continue"
+                "startText": "Begin",
+                "finalText": "Finish",
+                "_component": "trickle-button"
+            },
+            "_stepLocking": {
```

Comment:
these are all default values so don't need to be included, please could you remove the whole `_stepLocking` object?